rotkehlchen/api/rest.py

```diff
@@ -4353,6 +4354,22 @@ def get_evm_counterparties_details(self) -> Response:
             status_code=HTTPStatus.OK,
         )
 
+    def get_evm_products(self) -> Response:
+        """
+        Collect the mappings of counterparties to the products they list
+        """
+        ethereum_counterparties = self.rotkehlchen.chains_aggregator.ethereum.transactions_decoder.possible_products  # noqa: E501
```

Comment:
One idea. Instead having a variable per chain we could iterate over all evm transaction decoders in a loop. We already have a constant with evm chains that have a transactions decoder.

Reply:
This is not just a good to have idea. This is a must. Otherwise we will end up forgetting to add new stuff here each time a new evm chain is added. i.e. just now with polygon.

So in short: Iterate over all evm chains from the chain aggregator, and get their decoders and then get the possible products.